src/App.tsx

```diff
@@ -10,11 +10,18 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<h1 className="title">Home Page</h1>} />
+            <Route path="/home" element={<Navigate to="/" replace />} />
+            <Route path="/people" element={<PeoplePage />} />
+            <Route path="/people/:slug" element={<PeoplePage />} />
```

Comment:
The route '/people/:slug' uses the same component 'PeoplePage' as the '/people' route. Ensure that 'PeoplePage' can handle both cases appropriately, especially if ':slug' is expected to be used for dynamic content.